actors/builtin/reward/reward_actor.go

```diff
@@ -107,8 +107,8 @@ func (a Actor) LastPerEpochReward(rt vmr.Runtime, _ *adt.EmptyValue) *abi.TokenA
 
 func (a Actor) computePerEpochReward(st *State, clockTime abi.ChainEpoch, networkTime abi.ChainEpoch, ticketCount int64) abi.TokenAmount {
 	// TODO: PARAM_FINISH
-	newSimpleSupply := big.Rsh(big.Mul(SimpleTotal, taylorSeriesExpansion(clockTime)), FixedPoint)
-	newBaselineSupply := big.Rsh(big.Mul(BaselineTotal, taylorSeriesExpansion(networkTime)), FixedPoint)
+	newSimpleSupply := big.Rsh(big.Mul(SimpleTotal, taylorSeriesExpansion(clockTime)), FixedPoint-90)
```

Comment:
If you look at the state_test.go file, this is what its doing to get the expected values out.

Reply:
@davidad @zx that 90 looks pretty magic. Can we encapsulate whatever the Rsh is doing in a function?

Reply:
This is not how the function is meant to be used. The reason the test is doing this is to examine the 90 most significant bits of the output. Another way of phrasing it is that the test is like
```go
big.Rsh(big.Mul(2^90, taylorSeriesExpansion(x)),FixedPoint)
```
where 2^90 is taking the place of SimpleTotal. The reason it's 90 is that the dynamic range from attoFIL to a billion FIL is about 2^90, so the test is checking that it's exact to the appropriate precision (while leaving some slack for the implementation is changed, including the `FixedPoint` parameter).

Reply:
I have cleaned up and documented the test here: https://github.com/filecoin-project/specs-actors/pull/348/files

I don't know why or how newSimpleSupply is less than st.SimpleSupply, assuming that st.SimpleSupply is indeed the newSimpleSupply from the previous epoch. The minting function is monotonic. In any case, I am pretty confident this isn't the fix.